chef/cookbooks/ceph/libraries/default.rb

```diff
@@ -158,7 +160,7 @@ def get_osd_nodes()
     cluster_addr = ""
     public_addr = ""
 
-    public_addr = Chef::Recipe::Barclamp::Inventory.get_network_by_type(node, "admin").address
+    public_addr = Chef::Recipe::Barclamp::Inventory.get_network_by_type(node, "ceph_client").address
```

Comment:
This will likely break upgrade.. we need to think about what to do for Cloud6-> upgrade users (unless $bigwig decides that this is not important)